CHANGELOG.asciidoc

```diff
@@ -3,6 +3,112 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.6.0]]
+=== Beats version 7.6.0
+https://github.com/elastic/beats/compare/v7.5.1...v7.6.0[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Allow Metricbeat's beat module to read monitoring information over a named pipe or unix domain socket. {pull}14558[14558]
```

Comment:
I don't think this is a breaking change.

Reply:
No its not a breaking change its an enhacement.,

Reply:
Thanks.  I'll move it to the _Added_ section.

Reply:
@mikemadden42 Looks like this still needs to be moved?